frappe/public/js/frappe/utils/pretty_date.js

```diff
@@ -1,64 +1,71 @@
-// moment strings for translation
+function prettyDate(date, mini) {
+	if (!date) return '';
 
-function prettyDate(time, mini) {
-	if (!time) {
-		time = new Date();
-	}
-	if ('moment' in window) { // use frappe.ImportError ;)
-		let ret;
-		if (frappe.sys_defaults && frappe.sys_defaults.time_zone) {
-			ret = moment.tz(time, frappe.sys_defaults.time_zone).locale(frappe.boot.lang).fromNow(mini);
+	if (typeof (date) == "string")
+		date = new Date((date || "").replace(/-/g, "/").replace(/[TZ]/g, " ").replace(/\.[0-9]*/, ""));
+
+	let diff = (((new Date()).getTime() - date.getTime()) / 1000);
+	let day_diff = Math.floor(diff / 86400);
+
+	if (isNaN(day_diff) || day_diff < 0) return '';
+
+	if (mini) {
+		// Return short format of time difference
+		if (day_diff == 0) {
+			if (diff < 60) {
+				return __("Now");
```

Comment:
since everything is in lowercase everywhere, we'd like to maintain the same everywhere

```suggestion
				return __("now");
```

missed it last time around!